plugins/flowdock/lib/samson_flowdock/samson_plugin.rb

```diff
@@ -20,22 +21,47 @@ def flowdock_tokens
     def no_flowdock_token?(flowdock_attrs)
       flowdock_attrs['token'].blank?
     end
+
+    def enabled_flows_names
+      flowdock_flows.enabled.map(&:name)
+    end
+  end
+end
+
+Samson::Hooks.callback :deploy_defined do
+  Deploy.class_eval do
```

Comment:
It does.
But if the class does not exist it will create it. And I don't want that. I want to be sure I am redefining it. ```class_eval``` guarantees me that, since it will load the class first.